src/Features/Core/Portable/ExtractMethod/ExtractMethodResult.cs

```diff
@@ -57,18 +51,17 @@ internal class ExtractMethodResult
 
         internal ExtractMethodResult(
             OperationStatusFlag status,
-            IEnumerable<string> reasons,
+            ImmutableArray<string> reasons,
             Document? documentWithoutFinalFormatting,
             ImmutableArray<AbstractFormattingRule> formattingRules,
             SyntaxToken invocationNameToken,
             SyntaxNode? methodDeclarationNode)
         {
             Status = status;
 
-            Succeeded = status.Succeeded() && !status.HasSuggestion();
-            SucceededWithSuggestion = status.Succeeded() && status.HasSuggestion();
+            Succeeded = status.Succeeded();
```

Comment:
>Succeeded = status.Succeeded(); [](http://example.com/codeflow?start=12&length=31)

Just for verification, presence of suggestions no longer affects succeeded?

Reply:
nm, this is because of the way SucceededWithSuggestion was previously being called.

Reply:
correct.